webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/MeshController.scala

```diff
@@ -55,7 +59,14 @@ class MeshController @Inject()(
                                          organizationName: String,
                                          dataSetName: String,
                                          dataLayerName: String,
-                                         formatVersion: Int): Action[ListMeshChunksRequest] =
+                                         formatVersion: Int,
+                                         /* If targetMappingName is set, assume that meshfile contains meshes for
+                                           the oversegmentation. Collect mesh chunks of all *unmapped* segment ids
+                                           belonging to the supplied agglomerate id.
+                                           If it is not set, use meshfile as is, assume passed id is present in meshfile
+                                          */
```

Comment:
How does one get the meshes of mapped segment ids, without using the oversegmentation? Or is this handled somewhere else?

Reply:
If the selected meshfile is computed with a mapping, the passed id is assumed to be an agglomerate id. There is no difference here for this case. (or rather, it is expressed solely by which meshfile the frontend selects)

Reply:
Makes sense, thanks :+1: